shanoir-ng-studies/src/main/resources/application.yml

```diff
@@ -107,6 +107,9 @@ logging:
     org.springframework.web: ERROR
     org.hibernate: ERROR
 
+# This can be updated to set new servers available
```

Comment:
Dear Jean-Come, minor comments: OFSEP==https://shanoir-ofsep.irisa.fr might be missing + GME==https://shanoir.fli.ai, might be nice to be complet